app/services/credit_notes/validate_item_service.rb

```diff
@@ -53,9 +53,9 @@ def valid_fee?
       false
     end
 
-    # NOTE: Check if item amount is positive
+    # NOTE: Check if item amount is >= 0 (CreditNoteItem.amount_cents has numericality validation: greater_than_or_equal_to: 0)
```

Comment:
Ha! We both found this out independently, A fix for this line has already been merged :)

Reply:
nooooice
will cancel this change then 😁